arangod/Aql/AqlFunctionFeature.cpp

```diff
@@ -471,6 +471,7 @@ void AqlFunctionFeature::addMiscFunctions() {
   add({"PARSE_IDENTIFIER", ".", flags, &Functions::ParseIdentifier});
   add({"IS_SAME_COLLECTION", ".h,.h", flags, &Functions::IsSameCollection});
   add({"DECODE_REV", ".", flags, &Functions::DecodeRev});
+  add({"SHARD_ID", ".h|.", flags, &Functions::ShardId});
```

Comment:
The second parameter is not optional.
```suggestion
  add({"SHARD_ID", ".h,.", flags, &Functions::ShardId});
```